definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -4682,6 +4682,34 @@
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "02/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "PendingDA",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "02/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "PendingDA",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "02/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "PendingDA",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "02/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "PendingDA",
+    "UserRole": "caseworker-divorce-courtadmin",
```

Comment:
Do you not need to grant access to the event so that the caseworkers can actually run the 'PendingDA' event?

Reply:
Hi John, Yes